config/notify.yml

```diff
@@ -0,0 +1,2 @@
+api_key: <%= ENV['GOVUK_NOTIFY_API_KEY'] || "TEST_KEY"%>
+template_id: <%= ENV['GOV_UK_NOTIFY_TEMPLATE_ID'] || "80c6e214-8ed3-4437-bf57-79005ec74719" %>
```

Comment:
I think `template_id` is a required field when we call notify - I've hard coded the fallback for now, but I think we should probably add a default in puppet? thoughts @rubenarakelyan

Reply:
Would it not be better to fail if these env vars don't exist?

Reply:
Perhaps it would be nicer to have this file with a layout similar to [gov_delivery.yml](https://github.com/alphagov/email-alert-api/blob/master/config/gov_delivery.yml) with the environments.

Reply:
Can I play devil's advocate for a minute and ask if we actually have different data we want for these variables in test, dev and prod yet? 

If we don't then would it be a good idea to wait until we have separate distinct values per env before adding in the code to do this? We will never simplify it again if we add it in now and find we don't need it.

Reply:
API key - yes defo, so we don't accidentally send real-life emails in non-production environments

Template ID - can use the same one since Notify doesn't have the notion of different environments with different data